<launch>

    <node pkg="chassis_package" type="chassis_node" name="chassis_node" output="screen">
        <param name="port" type="string" value="/dev/ttyUSB0"/>
        <param name="pub_topic" type="string" value="/chassis/data"/>
        <param name="print_data" type="int" value="1"/>
        <param name="pub_frequency" type="int"  value="10"/>
    </node>
    
</launch>
